docs/src/modules/components/DemoLanguages.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import JSLogo from '@material-ui/docs/svgIcons/JSLogo';
+import TSLogo from '@material-ui/docs/svgIcons/TSLogo';
+import HookLogo from '@material-ui/docs/svgIcons/HookLogo';
+import { CODE_VARIANTS } from 'docs/src/modules/constants';
+
+function DemoLanguages(props) {
+  const { demo, gaEventCategory, onLanguageClick } = props;
+  const hasHooksVariant = demo.rawHooks;
+  const hasTSVariant = demo.rawTS;
+  const hasMultipleLanguages = hasHooksVariant || hasTSVariant;
+
+  if (!hasMultipleLanguages) {
+    return null;
+  }
+
+  return (
+    <>
```

Comment:
It's not consistent with the codebase where we have been exclusively using `<React.Fragment>`. There is one drawback with this syntax, we can't provide a key. What do you think of using `<React.Fragment>` now and that we open a discussion on the topic?